all/docs/api.md

```diff
@@ -761,6 +763,212 @@ Deletes an existing tenant.
 
 ---
 
+### Search tenants
+
+<access-level type="admin-or-readonly"/>
+
+::::: row
+:::: left
+
+Search tenant records within a parent tenant.
+
+<parameters path="/v0/tenants/{tenantId}/tenants/find" verb="post" />
+
+<br>
+
+#### Filter options
```

Comment:
Note to self (or whoever ports this): in DDG this is probably the sort of thing that should be its own component and/or "chunk of information".